examples/s3/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "s3"
+version = "0.1.0"
+authors = ["andy-thomason <andy@andythomason.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[workspace]
+
+[dependencies]
+rusoto_core = "0.45"
```

Comment:
Couldn't it be improved by using something like `rusoto_core = { path = "../../rusoto/core" }`?

Reply:
Yes. This may be a better idea.

The empty [workspace] is to prevent cargo from building the entire project.
However, if we make each example into a crate, we may need a version also.

```
rusoto_core = { path = "../../rusoto/core", version = "0.45 }
```

When you use both, `cargo publish` will re-write the toml to use just the version.

But for now, lets just go for the path.